server/src/controllers/Users/resolver.ts

```diff
@@ -4,44 +4,73 @@ import { prisma } from '../../prisma';
 
 import { UserWithInstanceRole } from '../../graphql-types';
 import { Permission } from '../../../../common/permissions';
+import { InstanceRoles } from '../../../prisma/generator/factories/instanceRoles.factory';
+import { getRoleName } from '../../util/chapterAdministrator';
+
+const instanceRoleInclude = {
+  instance_role: {
+    include: {
+      instance_role_permissions: {
+        include: { instance_permission: true },
+      },
+    },
+  },
+};
 
 @Resolver()
 export class UsersResolver {
   @Authorized(Permission.UsersView)
   @Query(() => [UserWithInstanceRole])
   async users(): Promise<UserWithInstanceRole[]> {
-    return await prisma.users.findMany({
+    const users = await prisma.users.findMany({
       orderBy: { name: 'asc' },
-      include: {
-        instance_role: {
-          include: {
-            instance_role_permissions: {
-              include: { instance_permission: true },
-            },
-          },
-        },
-      },
+      include: instanceRoleInclude,
+    });
+
+    // The chapter_administrator role is internal, so should not be displayed
+    const usersWithReplacedAdministrator = users.map((user) => {
+      if (user.instance_role.name !== InstanceRoles.chapter_administrator) {
+        return user;
+      }
+      const userWithReplacedRoleName = { ...user };
+      userWithReplacedRoleName.instance_role.name = InstanceRoles.member;
+      return userWithReplacedRoleName;
     });
+    return usersWithReplacedAdministrator;
   }
 
   @Authorized(Permission.UserInstanceRoleChange)
   @Mutation(() => UserWithInstanceRole)
   async changeInstanceUserRole(
-    @Arg('roleId', () => Int) roleId: number,
+    @Arg('roleName', () => String) roleName: string,
```

Comment:
```suggestion
    @Arg('roleName', () => String) newRole: string,
```
Same nitpick as before